test/e2e/driver-e2e-specs.js

```diff
@@ -35,4 +35,22 @@ describe('Driver', function () {
     });
     await driver.elementByName('Calculator');
   });
+
+  it('should set text to clipboard and paste', async function () {
+    await driver.init({
+      app: 'C:\\Windows\\System32\\notepad.exe',
+      platformName: 'Windows',
+      deviceName: 'WindowsPC'
+    });
+    const title = await driver.title();
+    title.should.eql('Untitled - Notepad');
+    const setClipboardTestText = 'Set Clipboard Test';
+    //Set test text
+    await driver.setClipboard(setClipboardTestText, 'plaintext');
```

Comment:
the text should be already base65 encoded:
```
await driver.setClipboard(Buffer.from(setClipboardTestText, 'utf8').toString('base64'), 'plaintext');
```

Reply:
https://github.com/appium/java-client/blob/5d6cf9de96a354e1efa1f77df2f20d25c7f4e0d8/src/main/java/io/appium/java_client/android/HasAndroidClipboard.java#L41

Reply:
But this documentation suggests driver.setClipboard should take a string?
https://github.com/appium/appium/blob/master/docs/en/commands/device/clipboard/set-clipboard.md
Example: await driver.setClipboard('happy testing', 'plaintext')

Reply:
for users it is a string, yes, but all clients must convert it to base64 representation in order to build the request json

Reply:
I don't know if WD does this properly (or if it implements such method at all %) )

Reply:
in such case you could step to the lower level and send the http request with the appropriate payload

Reply:
I just accepted that .setClipboard doesn't seem to convert a string to b64 as the documentation suggests, and implemented the b64 before passing